db/seeds.rb

```diff
@@ -1,7 +1,45 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'json'
+File.open("db/schools_data.json").each do |line|
+  parsed_course = JSON.parse(line) #Every line in a the .txt file represents an JSON course object with belonging time objects
+  department = parsed_course["department"]
+  grading_option = parsed_course["grading_option"]
+  # name = parsed_course["name"]
```

Comment:
Why isn't the course name part of the schema? We need that for the table we're displaying.

Reply:
Yeah, it was removed due to a request from Danny. Will add it back